boost-common/src/main/java/io/openliberty/boost/common/config/JDBCBoosterPackConfigurator.java

```diff
@@ -89,25 +149,54 @@ public void addServerConfig(Document doc) {
         serverRoot.appendChild(dataSource);
 
         Element jdbcDriver = doc.createElement(JDBC_DRIVER);
-        jdbcDriver.setAttribute("id", DERBY_EMBEDDED);
+        jdbcDriver.setAttribute("id", DERBY_EMBEDDED_DRIVER_REF);
         jdbcDriver.setAttribute(LIBRARY_REF, DERBY_LIB);
         serverRoot.appendChild(jdbcDriver);
     }
     
-    @Override
-    public Properties getServerProperties() {
+    private void addDb2Config(Document doc) {
     	
-    	return serverProperties;
-    }
+    	Element serverRoot = doc.getDocumentElement();
 
-    @Override
-    public String getDependency() {
+        // Find the root server element
+        NodeList list = doc.getChildNodes();
+        for (int i = 0; i < list.getLength(); i++) {
+            if (list.item(i).getNodeName().equals("server")) {
+                serverRoot = (Element) list.item(i);
+            }
+        }
 
-        return dependency;
-    }
+        // Add library
+        Element lib = doc.createElement(LIBRARY);
+        lib.setAttribute("id", DB2_LIB);
+        Element fileLoc = doc.createElement(FILESET);
+        fileLoc.setAttribute("dir", RESOURCES);
+        fileLoc.setAttribute("includes", "db2jcc*.jar"); 
+        lib.appendChild(fileLoc);
```

Comment:
So it's a directory but not everything is included... just `db2jcc*.jar`.    What's the intent here?  Are you supposed to supply a license somewhere (which would match this pattern)?

Reply:
@scottkurz - the jar is the only file that is part of the db2 maven dependency. Im not aware of a license.  is one typically needed?